docs/fsharp/tutorials/getting-started/getting-started-visual-studio.md

```diff
@@ -5,168 +5,147 @@ keywords: visual f#, f#, functional programming
 
-3. You can view the results either by pressing CTRL+F5 or by selecting a section of the code and then pressing ALT+ENTER.
-<br />
+## Using F# Interactive
 
+One of the best features of the Visual F# tooling in Visual Studio is the F# Interactive Window.  It allows you to send code over to a process which allows you to call that code and see the result interactively.
 
-### To create and use a class
+To begin, highlight the `square` function defined in your code.  Next, hold the **Alt** key and press **Enter**.  This executes the code in the F# Interactive Window.  You should see the F# Interactive Window appear with the following in it:
 
-1. The following code creates a **Person** class that has two properties, **Name** and **Age**. **Name** is a read-only property. Its value is immutable, as are most values in functional programming. You can create mutable values in F# if you need them, but you must explicitly define them as mutable. In the following class definition, the value of **Age** is stored in a mutable local variable, **internalAge**. The value of **internalAge** can be changed.
+```
+>
 
-[!code-fsharp[Main](../../../../samples/snippets/fsharp/contour/snippet10.fs)]
+val square : x:int -> int
 
-2. To test the class, declare two **Person** objects, make some changes, and display the results, as shown in the following code.
+>
+```
 
-[!code-fsharp[Main](../../../../samples/snippets/fsharp/contour/snippet11.fs)]
+This shows the same function signature for the `square` function, which which you saw earlier when you hovered over the function.  Because `square` is now defined in the F# Interactive process, you can call it with different values:
 
-The following lines are displayed.
+```
+> square 12;;
+val it : int = 144
+>square 13;;
+val it : int = 169
+```
 
-<br />  **Name:  John**
-<br />  **Age:   44**
-<br />  **Name:  Mary**
-<br />  **Age:   15**
-<br />  **False**
-<br />
+This executes the function, binds the result to a new name `it`, and displays the type and value of `it`.  Note that you must terminate each line with `;;`.  This is how F# Interactive knows when your function call is finished.  You can also define new functions in F# Interactive:
 
+```
+> let isOdd x = x % 2 <> 0;;
 
-### To view other examples in the F# tutorial
+val isOdd : x:int -> bool
 
-1. On the **File** menu, point to **New**, and then click **Project**.
-<br />
+> isOdd 12;;
+val it : bool = false
+```
 
-2. If you cannot see Visual F# in the **Templates Categories** pane, click **Other Languages**, and then click **Visual F#**. The **Templates** pane in the center lists the F# templates.
-<br />
+The above defines a new function, `isOdd`, which takes an `int` and checks to see if it's odd!  You can call this function to see what it returns with different inputs.  You can call functions within function calls:
```

Comment:
The above -> The previous code (Microsoft style guide tells us not to use above to mean earlier and suggest previous, preceding or earlier instead)